pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -117,16 +117,6 @@ type GraphBuilder struct {
 type monitor struct {
 	controller cache.Controller
 	store      cache.Store
-
-	// stopCh stops Controller. If stopCh is nil, the monitor is considered to be
-	// not yet started.
-	stopCh chan struct{}
-}
-
-// Run is intended to be called in a goroutine. Multiple calls of this is an
-// error.
-func (m *monitor) Run() {
-	m.controller.Run(m.stopCh)
```

Comment:
m.controller.Run(...) is still needed to be called somewhere, no?

Reply:
See comment below, this is actually a bug because the monitor's controller is actually a no-op.

Reply:
i think it's better fix the bug in a separate PR first, instead of mixing the fix inside this PR, what do you think?